assets/js/services/metadata.js

```diff
@@ -401,7 +402,19 @@ export function prepRelatedUrl(items = []) {
  * @param {Object} item
  * @returns {Object}
  */
-export function deleteKeyFromObject(item) {
+export function deleteKeyFromObject(item, key) {
+  if (key === "dateCreated") {
```

Comment:
Do we need this block of code?   It seems the function should be generic and accounting for a single value `dateCreated` within this function might not be the best spot for it, if the code block is really even needed.  I can't tell why we're calling it...

Reply:
Ah, now I see why that code is there.  Because of the API:

![image](https://user-images.githubusercontent.com/3020266/98837950-36f0fa00-2409-11eb-8c6b-b530502b3139.png)

I get it, but think we should move this unique scenario, which converts an array of string values, into an "edtf object" creator function.  

Maybe we code for that check in the `handleBatchEditConfirm()` function in the `Confirmation` component, and call it after we have a value for `cleanedPostValues`.